Hadrons/Modules/MScalar/ChargedProp.cpp

```diff
@@ -39,23 +40,18 @@ TChargedProp::TChargedProp(const std::string name)
 {}
 
 // dependencies/products ///////////////////////////////////////////////////////
-std::vector<std::string> TChargedProp::getInput(void)
+std::vector<std::string> TChargedProp::getInput()
 {
-    std::vector<std::string> in = {par().source, par().emField};
-    
-    return in;
+    return {par().source, par().emField};
 }
 
-std::vector<std::string> TChargedProp::getOutput(void)
+std::vector<std::string> TChargedProp::getOutput()
 {
-    std::vector<std::string> out = {getName(), getName()+"_0", getName()+"_Q",
-                                    getName()+"_Sun", getName()+"_Tad"};
-    
-    return out;
+    return {getName(), getName()+"_0", getName()+"_Q", getName()+"_Sun", getName()+"_Tad", getName()+"_projections"};
```

Comment:
We should keep `void` in the argument signature and keep `out` as a separate `std::vector<std::string>` for consistency with the rest of Hadrons here.